src/components/member-profile/load-more-btn/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import classNames from '@components/member-profile/load-more-btn/load-more-btn.module.scss';
+
+const LoadMoreButton = (props) => {
+  const { loadMoreHandler } = props;
+
+  return (
+    <div className={classNames.loadDiv}>
+      <button className={classNames.loadButton} onClick={loadMoreHandler}>
```

Comment:
We could remove line number 1 by specifying button type here right?
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/button-has-type.md

Reply:
What is the reason behind no array index key here?
Could this be looked into please